templates/element_doc_pages/any-page.html

```diff
@@ -0,0 +1,138 @@
+<style type="text/css">
+  blockquote {
+    background-color: black;
+    color: white;
+    padding: 0 1em;
+  }
+  h3 {
+    font-size: 1.2em;
+    font-weight: bold;
+    color: var(--primaryColor);
+  }
+</style>
+<h2>Adding an 'Any Page'</h2>
+
+<p>This example will demonstrate creating an arbitrary new page.</p>
+
+<p>It will list people and documents related to a specified term illustrating:</p>
+<ul>
+  <li>Template and query pairing</li>
+  <li>Use of existing web components</li>
+  <li>Use of custom css</li>
+</ul>
+
+<ol>
+  <li>
+    <h3>Define a Template - <small><em>templates/any_pages/spotlight_on.html</em></small></h3>
```

Comment:
Is it correct to gather that the names of `whatever.html` has to match `whatever.graphql`?

Reply:
Yes, they must match

Reply:
A comment to that affect my avoid future confusion.

Reply:
Sounds good, I'll work one in. Thanks!